contracts/NonFungibleToken-v2.cdc

```diff
@@ -51,6 +51,17 @@ access(all) contract NonFungibleToken {
     // An entitlement for allowing the withdrawal of tokens from a Vault
     access(all) entitlement Withdrawable
 
+    /// Event that is emitted when a token is updated,
+    ///
+    access(all) event Updated(id: UInt64, uuid: UInt64, owner: Address, type:String)
```

Comment:
Not sure if this is actually possible, but when you commented about it. I assumed that the event would include some sort of information about what was updated. Like an `AnyStruct`, a string description of what was updated, or a custom struct that we can define that includes multiple fields that elaborate on the update. Do you think something like that would be useful? I feel like with the way that the event currently is, indexers wouldn't get any useful information from this.

Reply:
What we do in the indexer for Doodles is that we just reevaluate all the views when this happends. We can always add a context map though.

Reply:
like you just query that owners account and check all the views for the doodle that was updated? I guess that works. idk if it would even be possible in a generic way to make that information accessible in an event

Reply:
We can discuss in the meeting next week